### DISCUSSION TITLE
Disable individual car from being logged

### DISCUSSION THREAD

**Discussion by tobiasehlert** [teslamate-org/teslamate#1049] [Ideas]
**What would you like to be added**:

I have access to some friends Tesla and it started loggin in my personal TeslaMate. I don't want my TeslaMate setup to log his car in addition to his own TeslaMate setup.

Therefore I think an option to disable logging for an specific car in setting by a toggling the car to on or off and remove data from dashboards and so on would be amazing!

**Why is this needed**:

Double logging increases the amount of requests sent to Tesla in addition to someone TeslaMate gets an additional car, when it's nothing of interest.

**Additional context**:

Is it possible to differentiate if the car is a shared car or an owned car? That could maybe a trigger to either be included into logging as well. Even if there are maybe people that like that the car gets logged from a friend/partner.

And maybe in Grafana it would nice to either choose which car to show, since a couple of views there show all cars at same time. That way we could "hide" a particular car from being shown as well.

**Comment by mathiasmora** [comment]
This feature would be awesome!

**Comment by brianjw** [comment]
Also interested in this functionality for the same reasons. Any workarounds available today - or plans to implement this functionality?

**Comment by romerod** [comment]
+1

**Comment by dagstuan** [comment]
I would also like to see this, since I currently have access to cars I have no interest in tracking. Maybe this can get some traction with new admins.

**Comment by elemated** [comment]
+1

**Comment by MrNickIE** [comment]
This request kind of overlaps with this one?
https://github.com/teslamate-org/teslamate/discussions/2719

**Reply by dagstuan** [reply]
I think that request is more for deleting an old car from the Teslamate database (like after selling it or for loaners). This request is to disable a vehicle from being tracked while keeping it in the Tesla app.

**Reply by DrMichael** [reply]
But it has a (temporary) solution in https://github.com/teslamate-org/teslamate/discussions/2719#discussioncomment-6472795

**Comment by mark3-dev** [comment]
I also needed the ability to disable individual cars on my Tesla account.
I created a fork here and added an enabled button to the settings page. https://github.com/mark3-dev/teslamate
I've been using this for a few weeks now and I'm happy with how it works.

If anyone wants to try it out I created a public docker image here ghcr.io/mark3-dev/teslamate:disable-cars

Does anyone know how I go about attempting to get this back into the main repo?

**Reply by cwanja** [reply]
Create a pull request, include your changes and request it be merged. It will be reviewed @mark3-dev

**Comment by jheredianet** [comment]
Is this what you are looking for?
https://github.com/teslamate-org/teslamate/pull/3993
